cookbook/integrations/kubernetes/pod/pod.py

```diff
@@ -49,65 +31,97 @@
     V1VolumeMount,
 )
 
-_SHARED_DATA_PATH = "/data/message.txt"
-
 # %%
-# We define a simple pod spec with two containers.
-
+# Simple example
+# ==============
+#
+# In this example, we define a simple pod specification.
+# The ``containers`` field is set to an empty list, signaling to Flyte to insert a placeholder primary container.
+# The ``node_selector`` field specifies the nodes on which the task pod should be run.
+@task(
+    task_config=Pod(
+        pod_spec=V1PodSpec(
+            containers=[],
```

Comment:
Why do we need this?

Reply:
`containers` cannot be `None` in `V1PodSpec`: https://github.com/kubernetes-client/python/blob/master/kubernetes/docs/V1PodSpec.md.